requirements.txt

```
@@ -3,6 +3,9 @@ setuptools==56.2.0
 wheel==0.36.2
 twine==3.4.1
 flake8==3.9.2
+pydantic==1.8.1
 pylint==2.8.2
 pytest==6.2.4
+pytest-asyncio==0.15.1
 pytest-cov==2.11.1
+python-jose==3.2.0
```

```suggestion
python-jose[cryptography]==3.2.0
```
-> https://pypi.org/project/python-jose/#description

---

When I tried this locally it did not work.

---

It's defined here: https://github.com/mpdavis/python-jose/blob/master/setup.py#L25
And the package exists: https://pypi.org/project/cryptography/
you might have done something wrong